### DISCUSSION TITLE
validation {validation}:{value}_if not working

### DISCUSSION THREAD

**Discussion by ruchit-parikh** [laravel/framework#33510] [General]
When I try to use validations like max:10_if:type,abc it's not working as it's supposed to be.
required_if works fine though.
I guess this issue only relates with validation which has one or more extra parameters separated by ':' or ','.

**Comment by AegirLeet** [comment]
`require_if` and `exclude_if` are specific rules, you can't just append the `_if` anywhere. You should take a look at [conditionally adding rules](https://laravel.com/docs/7.x/validation#conditionally-adding-rules) and [custom validation rule](https://laravel.com/docs/7.x/validation#custom-validation-rules).
